mobify/mobifyjs
# V1.1.1 - ADJS-87: Fix double mobify-path cookie

Removes creating the mobify-path=; cookie; mobify-path is set via preview.

Status: **DO NOT MERGE**\- this branch is for Chewy only until we know if this fixes their issue.

Reviewers: *\* @jansepar @tedtate @johnboxall**
JIRA: https://mobify.atlassian.net/browse/ADJS-87
## Changes
- removed cookie setting code from base.konf
## How to test-drive this PR
- Build yourself a mobify-client using this commit for mobify 1.1
- Build a bundle using the new client
- Preview the bundle and ensure that there's only one cookie set, and its value is `true`, not blank.
